data/json/items/tool_armor.json

```
@@ -2633,5 +2633,51 @@
     "revert_to": "foodperson_mask",
     "use_action": "FOODPERSON",
     "flags": [ "OUTER", "SUN_GLASSES", "NO_TAKEOFF", "LIGHT_10", "WATCH", "NO_UNLOAD" ]
+  },
+  {
+    "id": "attached_ear_plugs_on",
+    "type": "ARMOR",
+    "name": "pair of attached ear plugs (in)",
+    "name_plural": "pairs of attached ear plugs",
+    "description": "A pair of Industrial grade ear plugs attached together by some string. They're inside your ears, activate them to take them out.",
+    "weight": 22,
+    "volume": 0,
```

Maybe less - items should not have 0 volume (it's just bumped to 1 ml anyways).
```suggestion
    "volume": "10 ml",
```